monai/data/meta_tensor.py

```diff
@@ -82,10 +83,17 @@ def __new__(
         *args,
         **kwargs,
     ) -> MetaTensor:
-        return torch.as_tensor(x, *args, **kwargs).as_subclass(cls)  # type: ignore
+        _kwargs = {"device": kwargs.pop("device", None), "dtype": kwargs.pop("dtype", None)} if kwargs else {}
```

Comment:
If doing `device`, should we also do `dtype`?

Reply:
yes, the current pr supports both `device` and `dtype`

Reply:
Oh sorry I missed that!